marvin_actions.py

```diff
@@ -579,5 +579,7 @@ def marvinCommit(row):
     msg = None
     if any(r in row for r in ["commit", "-m"]):
         commitMsg = getCommit()
+        if commitMsg == "Du får komma på ett själv. Jag är trasig för tillfället!":
```

Comment:
maybe we could fix getCommit() so that it returns the right value to begin with?

Reply:
Something like this?

`
def getCommit():
    """
    Retrieves random commit message from whatthecommit.com/index.html
    """
    try:
        url = getString("commit", "url")
        r = requests.get(url, timeout=5)
        res = r.text.strip()
        msg = "Använd detta meddelandet: '{}'".format(res)
        return msg
    except Exception:
        return getString("commit", "error")

def marvinCommit(row):
    """
    Display a random commit message
    """
    msg = None
    if any(r in row for r in ["commit", "-m"]):
        commitMsg = getCommit()
    return commitMsg
`

Reply:
yes, this is good. if we were going to reuse getCommit() somewhere, we might want to have it only return the commit message, or throw an exception, and do the string formatting in marvinCommit(). That way getCommit() acts more like an API, and marvinCommit() defines the interaction with marvin. but i don't think that will ever happen, so we don't need to go that far.